packages/eas-cli/src/credentials/ios/utils/p12Certificate.ts

```diff
@@ -39,7 +39,23 @@ function getRawCertData(p12Buffer: Buffer | string, passwordRaw: string | null):
   const password = String(passwordRaw || '');
   const p12Der = forge.util.decode64(p12Buffer);
   const p12Asn1 = forge.asn1.fromDer(p12Der);
-  const p12 = forge.pkcs12.pkcs12FromAsn1(p12Asn1, password);
+
+  let p12: forge.pkcs12.Pkcs12Pfx;
+  try {
+    if (password) {
+      p12 = forge.pkcs12.pkcs12FromAsn1(p12Asn1, password);
+    } else {
+      p12 = forge.pkcs12.pkcs12FromAsn1(p12Asn1, false /* strict */);
+    }
+  } catch (_error) {
+    const error: Error = _error;
+    if (/Invalid password/.exec(error.message)) {
+      throw new Error('Provided password for the distribution certificate is probably invalid');
```

Comment:
```suggestion
      throw new Error('Provided password for the distribution certificate is invalid');
```

Why "probably" if the error message is explicitly saying it's invalid? :D

Reply:
https://github.com/expo/turtle-v2/blame/5cc2ad41356a1bbb783206a39f84e6b23eca0256/src/libs/build-tools/src/ios/credentials/distributionCertificate.ts

I think it's because malformed file can produce result like this

Reply:
```suggestion
      throw new Error('Could not parse distribution certificate. Check that your password is correct.');
```
^ This is what I've usually seen In products where they want to say 'the thing is probably not correct'. Even though it's pretty much saying the same thing, when users see the word 'probably', it makes the product seem less trustworthy 🤷‍♀️